semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +120,66 @@ p strong {
   margin-left: 10px;
 }
 
+/*--------------------------------
+   Application home & create
+---------------------------------*/
+#application-area {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+#application-area .top-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+#application-area .company-title {
+  width: 100%;
+  letter-spacing: @letterSpacingRegular;
+  font-family: @alternativeFont;
+  font-weight: @mildboldWeight;
+  line-height: @lineHeightBigger;
```

Comment:
Same comment about line height here. If you're just wanting space below, then `margin-bottom` is better. Line height only becomes relevant for multiple lines within a text block. And .company-title should never be more than one line.